zulip_bots/zulip_bots/bots/chess/chess.py

```diff
@@ -22,7 +23,7 @@ def usage(self):
             'Stockfish program on this computer.'
         )
 
-    def initialize(self, bot_handler):
+    def initialize(self, bot_handler: ExternalBotHandler) -> None:
```

Comment:
I think we should further apply this style to other bots as well. This is certainly loads better than `: Any`. @neiljp is this what you were recommending?